specs/007-rag-evaluation-framework/data-model.md

```
@@ -0,0 +1,192 @@
+    absolute_delta: float
+    percentage_change: float
+
+class ComparisonReport(BaseModel):
+    baseline_id: str
+    current_id: str
+    deltas: dict[str, MetricDelta]
+```
+
+## Entity Relationships
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)